fendermint/vm/topdown/proof-service/src/service.rs

```diff
@@ -95,169 +116,228 @@ impl ProofGeneratorService {
             cache,
             f3_client,
             assembler,
+            pending_certificate: None,
```

Comment:
Should be the one with instance ID `start_instance`, but perhaps we don't need `pending_certificate` field in `ProofGeneratorService`.